src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -975,6 +976,17 @@ def set_up_runtime(self):
                         "java": runtime_java_def
                     }
             safe_set(self.containerapp_def, "properties", "configuration", "runtime", value=runtime_def)
+    
+    # pylint: disable=unsupported-assignment-operation
+    def set_up_scale_rule(self):
```

Comment:
Should we add validation logic about the `--scale-rule-identity` is only for a custom scale rule?